MapboxNavigation/Constants.swift

```diff
@@ -1,6 +1,35 @@
 import Foundation
+import MapboxDirections
+
+typealias CongestionSegment = ([CLLocationCoordinate2D], CongestionLevel)
+
+/**
+ Line width base values used at zoom levels.
+ */
+public let lineWidthAtZoomLevels: [Int: MGLStyleValue<NSNumber>] = [
```

Comment:
I don’t think we should expose this constant publicly at all. The whole point of making the route line style customizable is that the developer would substitute this style value with their own.